docs/attributes-registry/error.md

```diff
@@ -9,9 +9,12 @@ This document defines the shared attributes used to report an error.
 
 | Attribute | Type | Description | Examples | Stability |
 |---|---|---|---|---|
-| <a id="error-type" href="#error-type">`error.type`</a> | string | Describes a class of error the operation ended with. [1] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| <a id="error-message" href="#error-message">`error.message`</a> | string | A message providing more detail about an error in human-readable form. [1] | `TypeError: Cannot read properties of undefined (reading 'toString')` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="error-type" href="#error-type">`error.type`</a> | string | Describes a class of error the operation ended with. [2] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 
-**[1] `error.type`:** The `error.type` SHOULD be predictable, and SHOULD have low cardinality.
+**[1] `error.message`:** `error.message` should provide additional context and detail about an error. It is NOT RECOMMENDED to duplicate the value of `error.type` in `error.message`.
```

Comment:
Can we say that this attribute would usually be used on the logs/events and should not be captured on spans where the same information is in the status description? also not captured on metrics due to cardinality.

It could also be useful to add a link to https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/recording-errors.md

WDYT?

Reply:
what do you think of the wording in 93ec7cf1cf7da521be0e5294b4180bec29075976